src/css/imports/mixins.less

```diff
@@ -1,9 +1,10 @@
 @import "vars";
 
 .inset-controlbar() {
+    display: block;
     margin: 0 auto;
-    max-width: @controlbar-max-width;
-    bottom: @ui-margin;
+    max-width: 96%;
```

Comment:
Hmm don't we want to use margins and a fixed max width? percentages tend to be arbitrary / last ditch effort.